maro/cli/inspector/params.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+
+
+class GlobalFilePaths:
+    ports_sum = "ports_summary.csv"
+    vessels_sum = "vessels_summary.csv"
+    stations_sum = "stations_summary.csv"
+    name_convert = "name_conversion.csv"
+
+
+class GlobalScenarios(Enum):
+    CIM = 1
```

Comment:
use string instead of int:

CIM="Container Inventory Management"
CITI_BIKE="Citi Bike"

Or some other names? @ArthurJiang

Reply:
According to the argparse in maro/cli/inspector/visualization.py,
I think "cim" and "citi_bike" are better